Products/ZenRRD/zencommand.py

```diff
@@ -491,11 +494,14 @@ def doTask(self):
 
         except Exception as e:
             self.state = TaskStates.STATE_PAUSED
-            err_msg = "Task paused  name=%s device-id=%s ip=%s message=%s" % (
-                self.name,
-                self._devId,
-                self._manageIp,
-                e.message)
+            err_msg = (
```

Comment:
Do we need an outer tuple for err_msg here?

Reply:
It's not really a tuple.  There's no trailing comma.  E.g. `()` is a tuple, `(1,)` is a tuple, but `(1)` is not a tuple.  It's just an expression. Python removes the surrounding parenthesis.

The alternative was to use to a backslash, but I don't like how they look.  Using parenthesis to break up a statement onto multiple lines has a better aesthetic, imho.  Here's what the alternative would look like:
```
        err_msg = \
            "Task paused  device=%s ip=%s interval=%s message=%s" % (
                self._devId,
                self._manageIp,
                self.interval,
                e.message,
            )
```